### ISSUE TITLE
A few comments on undervolting

### ISSUE THREAD

**Issue by derlhurgoyf** [tom-james-watson/dotfiles#1]
Hi, I hope it's ok to give you some feedback on the notes you posted.

I've applied most of your suggestions on my X1 Extreme on Kubuntu 19.04. However when running a CPU test with primesieve the suggested undervolt parameters caused a crash, so I set them to half their values.
I also noticed that for some strange reason the CPU temperature cap is getting reset to 80° after a while and only the time service will reset it. This seems a bit odd.
I've also read on a few pages where people suggest repasting the CPU for better temperature. I'll try that as well and get back with the results.

**Comment by tom-james-watson** [comment]
Yeah, feedback is most welcome! I ran those settings for a long time without any stability, so I guess what is stable will depend on the thermal performance of your particular machine.

I'd be interested to see the results of repasting, so please share if you do that!
